src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  color: $main-text-color;
+  background-color: #fff;
+  border: 1px solid $light-grey-color;
+  padding: 32px 16px 16px;
+  transition: $hover-duration;
+
+  &__img {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 0 4px;
+    transition: $hover-duration;
```

Comment:
The transition property uses `$hover-duration`. Make sure this variable is defined and imported correctly in your SCSS setup.